MDANSE_GUI/Src/MDANSE_GUI/Tabs/Models/JobHolder.py

```diff
@@ -246,8 +270,19 @@ def startProcess(self, job_vars: list):
         communicator = JobCommunicator()
         watcher_thread = JobThread(communicator, main_pipe, subprocess_ref)
         communicator.moveToThread(watcher_thread)
-
+        entry_number = self.next_number
+        item_th = JobEntry(
+            command=job_vars[0],
+            entry_number=entry_number,
+            pause_event=pause_event,
+            load_afterwards=load_afterwards,
```

Comment:
Just noticed one more thing, the logs are no longer outputted to the job running tab. The first job entry above is the one that has all the logs. I think we need to merge the two JobEntry objects.